packages/airnode-deployer/src/infrastructure/index.ts

```diff
@@ -248,6 +259,7 @@ export async function terraformAirnodeApply(
   const cloudProvider = config.nodeSettings.cloudProvider as CloudProvider;
   const airnodeAddress = deriveAirnodeAddress(airnodeWalletMnemonic);
   const airnodeWalletPrivateKey = evm.getAirnodeWallet(config).privateKey;
+  logger.setSecret(airnodeWalletPrivateKey);
```

Comment:
I went with this option to set secret values to be sanitized from logs. The deployer command options are passed down multiple levels and I would have needed to pass the list of secret values along for them to reach the logger call. So this seemed simpler.